src/ontology/Makefile

```diff
@@ -41,7 +41,7 @@ REPORT_PROFILE_OPTS =       --profile $(ROBOT_PROFILE)
 OBO_FORMAT_OPTIONS =        
 SPARQL_VALIDATION_CHECKS =  properties-as-annotation-and-object ordo-mapping-annotations 
 SPARQL_EXPORTS =            basic-report 
-ODK_VERSION_MAKEFILE =      v1.3.1
+ODK_VERSION_MAKEFILE =      v1.3.0
```

Comment:
you have to run: `docker pull obolibrary/odkfull` before running update_repo - you are downgrading the odk here!

Reply:
Whoops. Can we add that docker command as a dependency or first line in `update_repo`? I wouldn't be a surprised if I (or possibly someone else) accidentally did this again at some point

Reply:
Maybe we could add a -U option to the run.sh command to automatically upgrade, but doing it by default has been decided against because of the possible breaking changes..

Reply:
Hmmm. I see what you're saying. I will try to remember, then.